src/app/course-page/components/course-item/course-item.component.ts

```diff
@@ -31,13 +34,23 @@ export class CourseItemComponent implements OnInit, OnChanges {
   }
 
   public deleteCourseItem(): void {
-    this.onDelete.emit(this.courseItem);
+    if (this.auth.getAutorizationValue()) {
+      let confirmation = prompt("Do you really want to delete this course? Yes/No", "");
+      if (confirmation.toLowerCase() === "yes") {
+        this.onDelete.emit(this.courseItem);
+      } else {
+        console.log("This course won't be deleted", confirmation);
+      }
+    }
   }
 
   public updateCourseItem(): void {
-    if (this.auth.getAutorizationValue()) {
-      this.itemService.rememberId(this.courseItem.id);
-    }
+    // if (this.auth.getAutorizationValue()) {
+    //   this.itemService.rememberId(this.courseItem.id);
+    // }
+
+    this.itemService.rememberId(this.courseItem.id);
+    this.loadService.loadingBlock();
```

Comment:
method should have a verb in the name, cause method is an action. Please, rename it to smth like "showLoad"

Reply:
changed naming of the method